css/style.css

```diff
@@ -287,6 +303,21 @@ h4 {
   font-weight: bold;
 }
 
+.wide-screen-view-modal {
+  display: none;
+}
+
+.tonic-see-live:hover,
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you add the correct different button interactions including hover, active (when it's pressed) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

Reply:
Thank you for your comment. I believe I have done as per the design. would you please elaborate on your suggestion so that I make changes? thank you

Reply:
Hi @emmanuellmhango , Sorry for the late reply, you used the active color for hover. Please use the correct colors.